ddp-workspace/projects/ddp-testboston/src/app/app-routing.module.ts

```diff
@@ -23,13 +23,6 @@ import {
 } from 'toolkit';
 
 const routes: Routes = [
-  {
-    path: AppRoutes.Prequalifier,
-    component: WorkflowStartActivityRedesignedComponent,
-    canActivate: [
-      IrbGuard
-    ]
-  },
```

Comment:
Instead of `WorkflowStartActivityRedesignedComponent` we use `UserRegistrationPrequalComponent`